src/pages/gateway/fastly.md

```diff
@@ -0,0 +1,251 @@
+---
+title: Fastly 
+description: Learn how to configure a Fastly CDN for Adobe Commerce using API Mesh.
+keywords:
+  - API Mesh
+  - Extensibility
+  - Cache
+  - GraphQL
+  - Integration
+  - REST
+  - Tools
+---
+
+# Fastly configuration
+
+Using [Fastly](https://experienceleague.adobe.com/docs/commerce-cloud-service/user-guide/cdn/fastly.html?lang=en) as a content delivery network (CDN) with API Mesh for Adobe Developer App Builder provides additional security and improved performance. To fully integrate API Mesh, Fastly, and Adobe Commerce, you must follow the configuration instructions in the following sections.
+
+## Configure Fastly in Adobe Commerce
+
+Use the following process to configure edge caching for Adobe Commerce using the provided Fastly CDN.
+
+1. [Get your Fastly credentials](https://experienceleague.adobe.com/docs/commerce-cloud-service/user-guide/cdn/setup-fastly/fastly-configuration.html?lang=en#get-fastly-credentials).
+
+1. In the Adobe Commerce Admin, select **Store** > Settings > **Configuration** > **Advanced** > **System** > **Full Page Cache** and set the **Caching Application** field to **Fastly CDN**.
+
+1. Enter the Fastly credentials you retrieved previously into the **Fastly Service ID** and **Fastly API Token** fields. Then click the Test Credentials button to verify that your credentials are correct.
+
+1. Under **Fastly Configuration** > **Custom VCL Snippets** click **Create** and add the following [snippets](https://experienceleague.adobe.com/docs/commerce-cloud-service/user-guide/cdn/custom-vcl-snippets/fastly-vcl-custom-snippets.html). For more information on VCL subroutines, see [Custom Subroutines](https://developer.fastly.com/reference/vcl/subroutines/).
```

Comment:
no, because you can't modify that with the UI - it hides the custom VCL that's generated by Fastly module 
SO there's no way to update..
we also want to prevent patching code - which would be the other way to accomplish this- so we will use new Custom VCL Snippets for all modifications and play with the priority to get around existing VCL to overriding existing logic